src/analyzer/protocol/ssl/SSL.cc

```diff
@@ -8,6 +9,73 @@
 #include "zeek/analyzer/protocol/ssl/ssl_pac.h"
 #include "zeek/analyzer/protocol/ssl/tls-handshake_pac.h"
 
+#include <arpa/inet.h>
+#include <openssl/evp.h>
+#include <openssl/kdf.h>
+
+#define MSB(a) ((a>>8)&0xff)
+#define LSB(a) (a&0xff)
+
+static void fmt_seq(uint32_t num, u_char* buf)
+	{
+	memset(buf, 0, 8);
+	uint32_t netnum = htonl(num);
+	memcpy(buf+4, &netnum, 4);
+	}
+
+static void print_hex(std::string name, u_char* data, int len)
+	{
+	int i = 0;
+	printf("%s (%d): ", name.c_str(), len);
+	if (len > 0)
+		printf("0x%02x", data[0]);
+
+	for (i = 1; i < len; i++)
+		{
+		printf(" 0x%02x", data[i]);
+		}
+	printf("\n");
+	}
+
+static bool tls1_prf(const u_char *secret, int secret_len, const char* label,
```

Comment:
This probably should be a member function of the analyzer - unless there is a specific reason why it shouldn't be.